pkgs/tools/admin/awscli2/default.nix

```diff
@@ -72,6 +72,8 @@ with py.pkgs; buildPythonApplication rec {
     mkdir -p $out/share/zsh/site-functions
     mv $out/bin/aws_zsh_completer.sh $out/share/zsh/site-functions
     rm $out/bin/aws.cmd
+    PYTHONPATH=${src}:$PYTHONPATH
+    ${src}/scripts/gen-ac-index --index-location $out/lib/python3.8/site-packages/awscli/data/ac.index
```

Comment:
```suggestion
    ${src}/scripts/gen-ac-index --index-location $out/${python3.sitePackages}/awscli/data/ac.index
```

Reply:
this is mainly important because if python version changes the build will fail

Reply:
Thanks, I thought there was probably a better way of doing that.